typescript/src/codeActions/custom/renameToTypeParameter.ts

```diff
@@ -0,0 +1,53 @@
+import { pipe, groupBy, map } from 'lodash/fp'
+import { CodeAction } from '../getCodeActions'
+import extractType from '../../utils/extractType'
+
+export default {
+    id: 'renameToTypeParameter',
+    name: 'Rename to Type Parameter',
```

Comment:
```suggestion
    name: 'Rename Parameter to Type Name',
```
not ideal, but much better. type parameters in ts refers to `type Foo<K, L> = ...`